tools/commit_slider/case_1.md

```diff
@@ -0,0 +1,316 @@
+# Case #1. Cross-checking of performance degradation.
+#### Themes: Mode and Template customization, API design
+  
+
+For 2 given versions, defined by commit hashes and 2 models we need to resolve which of two changes (ov or model) caused degradation. I.e. we need to differentiate 2 possible cases:
+model-caused
+|'Bad' model | model_1.xml| model_2.xml|
+| --------     | -------    | -------|
+| ov_version_1 | 100 % (ref)|50%     |
+| ov_version_2 | 98 %       |51%     |
+and ov - caused
+|'Bad' ov-version| model_1.xml| model_2.xml|
+| --------     | -------    | -------|
+| ov_version_1 | 100 % (ref)|98%     |
+| ov_version_2 | 48% %       |51%     |
+
```

Comment:
Can we add a few introductory words regarding what we will be considering next?
As far as I understand, in this file we consider several scenarios (from simple to complex), in which we try to solve the above problem using the commit slider

Reply:
Done.